pkgs/initiator/initiator.go

```diff
@@ -280,8 +293,103 @@ func (c *Initiator) StartDKG(id [24]byte, withdraw []byte, ids []uint64, network
 	return depositDataJson, keyshares, proofsArray, nil
 }
 
+func (c *Initiator) StartResigning(id [24]byte, ids []uint64, proofs []*spec.SignedProof, network eth2_key_manager_core.Network, withdraw []byte, owner [20]byte, nonce uint64) (*wire.DepositDataCLI, *wire.KeySharesCLI, []*wire.SignedProof, error) {
+	if len(proofs) == 0 {
+		return nil, nil, nil, fmt.Errorf("🤖 unmarshaled proofs object is empty")
+	}
+	ops, err := ValidatedOperatorData(ids, c.Operators)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	pkBytes, err := spec_crypto.EncodeRSAPublicKey(&c.PrivateKey.PublicKey)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	instanceIDField := zap.String("Ceremony ID", hex.EncodeToString(id[:]))
+	c.Logger.Info("🚀 Starting dkg ceremony", zap.String("initiator public key", string(pkBytes)), zap.Uint64s("operator IDs", ids), instanceIDField)
+	// make resign message
+	rMsg := &wire.ResignMessage{
+		Operators: ops,
+		Resign: &spec.Resign{ValidatorPubKey: proofs[0].Proof.ValidatorPubKey,
+			Fork:                  network.GenesisForkVersion(),
```

Comment:
PR was created before the fork. Now we need to incorporate fork.

Reply:
For now I think it can be genesis

```
// GenesisForkVersion returns the genesis fork version of the network.
func (n Network) GenesisForkVersion() phase0.Version {
	switch n {
	case PyrmontNetwork:
		return phase0.Version{0, 0, 32, 9}
	case PraterNetwork:
		return phase0.Version{0x00, 0x00, 0x10, 0x20}
	case HoleskyNetwork:
		return phase0.Version{0x01, 0x01, 0x70, 0x00}
	case MainNetwork:
		return phase0.Version{0, 0, 0, 0}
	default:
		logrus.WithField("network", n).Fatal("undefined network")
		return phase0.Version{}
	}
}
```